frontend/src/features/User/UserMenu.tsx

```diff
@@ -0,0 +1,40 @@
+import { ChangeEvent, useCallback } from 'react';
+import  { Avatar, Button, HStack, VStack, Text, Flex, useHover, Tooltip, Popover, PopoverTrigger, PopoverContent, PopoverHeader, PopoverBody } from "@yamada-ui/react"
+import { BsThreeDots } from "react-icons/bs";
+import { useRouter } from "next/router";
+
+export const UserMenu: React.FC = () => {
+  const router = useRouter()
+
+  const handleClick = useCallback(() => {
+    // ここにクリックされたときの処理を記述します
+    console.log("HStack clicked"); 
+  }, []);
+
+  const logout = useCallback(() => {
+    router.push('/logout');
+  }, [router])
+  
+  const { hovered, ref } = useHover()
+
+  return (
+    <Popover placement="top">
+      <PopoverTrigger>
+          <Button lineHeight={"1.3"}>
```

Comment:
variantにunstyledを付けると初期の背景色消える

```suggestion
          <Button variant="unstyled" lineHeight={"1.3"}>
```